analyzers.yml

```
@@ -148,3 +148,7 @@ analyzers:
     name: TyScan
     github: sider/TyScan
     doc: tools/javascript/tyscan
+  remark:
+    name: remarklint
```

[suggestion] 

- ID: `remark` -> `remark_lint`
  - Because `remark` and `remark-lint` are different and may be confusing, and
  - IDs allow only `[a-z]` and `_` chars. See the [document](https://github.com/sider/runners/blob/3f385a0595a4e8b779ee4824f1b5d06cbd112cc3/docs/how-to-write-a-new-runner.md#define-a-runner-id).
- Name: `remarklint` -> `remark-lint`
  - Because the official name seems `remark-lint` not `remarklint`, and
  - the naming consistency will prevent us from confusing.

```suggestion
  remark_lint:
    name: remark-lint
```